vfio-bindings/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "vfio-bindings"
+version = "0.0.1"
+authors = ["The Cloud Hypervisor Authors"]
+edition = "2018"
+
+[dependencies]
+vmm-sys-util = { git = "https://github.com/rust-vmm/vmm-sys-util" }
+
+[features]
+default = ["v5_0_0"]
```

Comment:
I note that default features are a bit sticky. See https://github.com/rust-lang/cargo/issues/3126#issuecomment-250300450 for some background but if we want people to be able to selectively disable these features it may be better to note make it a default.